### PR TITLE
review typos + format

### PR DESCRIPTION
**Pull request by Gau-thier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,15 +59,15 @@ type storage =
   };
 ```
 
-Explanations :
+Explanations:
 
-- `offers` is now a `map<address,bid>`, is because you don't have to store token_id as key, now the key is the owner address. Each owner can sell a part of the unique collection
+- `offers` is now a `map<address,bid>`, because you don't have to store `token_id` as key, now the key is the owner address. Each owner can sell a part of the unique collection
```
Comment (Gau-thier) [inline README.md:64]: Is it `map<address,bid>`?

File: README.md [hunk 2]
```diff
@@ -59,15 +59,15 @@ type storage =
...
 - `offer` requires a quantity, each owner will sell a part of the unique collection
-- `totalSupply` can be used to track the glocal quantity of minted collection. It avoid to recalculates all the time the quantity of each holder (this does not move so we can store this value at mint time)
-- Because the ledger is made of big map of key `owners`, we cache the keys to be able to loop on it
+- `totalSupply` in order to track the global quantity of minted collection. It avoids to recalculate all the time the quantity of each holder (this does not move so we can store this value at mint time)
```
Comment (Gau-thier) [inline README.md:66]: Please rework this sentence, I am not sure to fully understand it.

File: README.md [hunk 3]
```diff
@@ -59,15 +59,15 @@ type storage =
...
+- Because the ledger is made of `big_map` of key `owners`, we cache the keys to be able to loop on it
```
Comment (Gau-thier) [inline README.md:67]: Should this technical detail be explained? Moreover, I don't see the related code...

File: README.md [hunk 4]
```diff
@@ -59,15 +59,15 @@ type storage =
...
 - we remove `token_ids` has we have an unique collection, token_id will be set to `0`
```
Comment (Gau-thier) [inline README.md:68]:
```suggestion
- Since we have a unique collection,we remove `token_ids`. `token_id` will be set to `0`
```

File: README.md [hunk 5]
```diff
@@ -293,7 +291,7 @@ const refreshUserContextOnPageReload = async () => {
 
 ## Update in `MintPage.tsx`
 
-We introduce the quantity, and remove the token_id variable. Replace the full file
+We introduce the quantity, and remove the `token_id` variable. Replace the full file by the following content:
```
Comment (Gau-thier) [inline README.md:294]:
```suggestion
We introduce the quantity and remove the `token_id` variable. Replace the full file by the following content:
```

File: README.md [hunk 6]
```diff
@@ -727,7 +725,7 @@ export default function MintPage() {
 
 ## Update in `OffersPage.tsx`
 
-We introduce the quantity, and remove the token_id variable. Replace the full file
+We introduce the quantity, and remove the `token_id` variable. Replace the full file by the following content:
```
Comment (Gau-thier) [inline README.md:728]:
```suggestion
We introduce the quantity and remove the `token_id` variable. Replace the full file by the following content:
```

File: README.md [hunk 7]
```diff
@@ -1049,7 +1047,7 @@ export default function OffersPage() {
 
 ## Update in `WineCataloguePage.tsx`
 
-We introduce the quantity, and remove the token_id variable. Replace the full file
+We introduce the quantity, and remove the `token_id` variable. Replace the full file by the following content:
```
Comment (Gau-thier) [inline README.md:1050]:
```suggestion
We introduce the quantity and remove the `token_id` variable. Replace the full file by the following content:
```

File: README.md (hunk 1 above)
Comment (zamrokk) [reply README.md:64]: map<address,offer>
I forgot this old refacto

File: README.md (hunk 2 above)
Comment (zamrokk) [reply README.md:66]: done. To know how much is minted, either to recalculate frol holder holdings, or you store this value once at mint time

File: README.md (hunk 3 above)
Comment (zamrokk) [reply README.md:67]: on mint function: `totalSupply: quantity,`

**Pull request merged**